server/integration/testsuite/src/test/java/org/infinispan/persistence/cluster/MyCustomCacheStore.java

```diff
@@ -8,6 +8,10 @@
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
```

Comment:
hmm this is a fail condition and because we use TestNG and JUnit in Infinispan, we have at least 4 options:
* throw AssertionError (universal, but you asked me to refactor it :D)
* TestNG/JUnit specific fail method (clearly defines that this is execution branch should fail)
* Standard java assertions (universal but needs running java process with -ea)
* Use other library which abstracts assertions (like AssertJ, but it needs an agreement from whole the team).

I'm a big fan of option number 4 (this allows to avoid problems with assertEquals in JUnit/TestNG - the order of arguments in opposite) but it needs more discussion. I think in this specific case option number 2 fits the best.